src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -77,6 +83,28 @@ public Response getCollectionByReferenceId(
     }
   }
 
+  @GET
+  @Path("query")
+  @Produces("application/json")
+  @RolesAllowed(RESEARCHER)
+  public Response getCollectionsByInitialQuery(
+    @Auth AuthUser authUser,
+    @QueryParam("filterTerm") String filterTerm,
+    @QueryParam("sortField") String sortField,
+    @QueryParam("sortOrder") String sortOrder,
+    @QueryParam("pageSize") int pageSize
+  ) {
+    try {
+      User user = userService.findUserByEmail(authUser.getEmail());
+      checkIfSortFieldAndSortOrderIsPresent(sortField, sortOrder);
```

Comment:
Can we instead pick a default like `dar code` `ascending` instead of throwing a bad request? That way the user will get something back and can re-query or re-sort depending on preference.